File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {

	public List<TbBrand> findAllBrands();
	
	public PageResult findPageBrands(int pageNum,int pageSize);
	
	public void addBrand(TbBrand brand);
	
	public TbBrand findOne(long id);
	
	public void update(TbBrand brand);
	
	public void delete(Long[] ids);
	
	public PageResult findPage(TbBrand brand,int pageNum,int pageSize);
	
	public List<Map> selectBrandOptionList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService", function($http) {

	this.findOne = function(id) {
		return $http.get('../brand/findOne.do?id=' + id);
	}

	this.dele = function(ids) {
		return $http.get('../brand/delete.do?ids=' + ids);
	}

	this.add = function(entity) {
		return $http.post('../brand/add.do', entity);
	}

	this.update = function(entity) {
		return $http.post('../brand/update.do', entity);
	}

	this.search = function(pageNum, pageSize, searchEntity) {
		return $http.post('../brand/search.do?pageNum=' + pageNum
				+ "&pageSize=" + pageSize, searchEntity);
	}
	
	//
	this.selectBrandOptionList=function(){
		return $http.get('../brand/selectBrandOptionList.do');
	} 
})